performance/framework/build.gradle

```diff
@@ -0,0 +1,115 @@
+buildscript {
+    ext.rootBuildDirectory = file('../..')
+
+    apply from: "$rootBuildDirectory/gradle/loadRootProperties.gradle"
+    apply from: "$rootBuildDirectory/gradle/kotlinGradlePlugin.gradle"
+
+    repositories {
+        maven {
+            url 'https://cache-redirector.jetbrains.com/jcenter'
+        }
+        maven {
+            url kotlinCompilerRepo
+        }
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+    }
+}
+
+apply plugin: 'kotlin-multiplatform'
+
+repositories {
+    maven {
+        url 'https://cache-redirector.jetbrains.com/jcenter'
+    }
+    maven {
+        url kotlinCompilerRepo
+    }
+    maven {
+        url buildKotlinCompilerRepo
+    }
+}
+
+def toolsPath = '../../tools'
+def frameworkName = 'benchmarksAnalyzer'
+
+kotlin {
+    sourceSets {
+        macosMain {
+            dependencies {
+                implementation "org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion"
+            }
+            kotlin.srcDir "$toolsPath/benchmarks/shared/src"
+            kotlin.srcDir "$toolsPath/benchmarksAnalyzer/src/main/kotlin"
+            kotlin.srcDir "$toolsPath/kliopt"
+            kotlin.srcDir "$toolsPath/benchmarksAnalyzer/src/main/kotlin-native"
+        }
+    }
+
+    configure([macosX64("macos")]) {
+        compilations.main.cinterops {
+            libcurl {
+                defFile "$toolsPath/benchmarksAnalyzer/src/nativeInterop/cinterop/libcurl.def"
+            }
+        }
+    }
+
+    macosX64("macos").binaries {
+        framework(frameworkName, [RELEASE])
+    }
+}
+
+MPPTools.addTimeListener(project)
+
+task konanRun {
+    if (MPPTools.isMacos()) {
+        dependsOn 'build'
+    }
+}
+
+task jvmRun(type: RunJvmTask)  {
+    doLast {
+        println("JVM run is unsupported")
+    }
+}
+
+task konanJsonReport {
```

Comment:
Shalln't we use .kts for the new non-trivial code?

Reply:
It isn't fully separated file, it's part of performance project. To use Kotlin in Gradle build files, I have to rewrite full project. I have already done such attempt, but there were problems with current project.